projects/minizinc/minizinc_fuzzer.cpp

```diff
@@ -0,0 +1,48 @@
+#include <minizinc/solver.hh>
+
+using namespace std;
+using namespace MiniZinc;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+
+    Timer starttime;
+    bool fSuccess = false;
+    try {
+      MznSolver slv(std::cout,std::cerr);
+      try {
+        std::string model = std::string(reinterpret_cast<const char*>(Data), Size);
+        std::vector<std::string> args({"-c","--solver","org.minizinc.mzn-fzn"});
+        fSuccess = (slv.run(args, model, "minizinc", "model.mzn") != SolverInstance::ERROR);
+      } catch (const LocationException& e) {
+        if (slv.get_flag_verbose())
```

Comment:
This wont log heavily when run on OSS-Fuzz right?

Reply:
This is code copied form the main MiniZinc executable. When it is executed in verbose mode extra line breaks are added. The fuzzer, however, is never executed in verbose mode.
The only output that the fuzzer should give are the errors produced by the compiler whenever an error is found.